<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelSetCode</class>
 <widget class="QWidget" name="SelSetCode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(27, 150, 208);
}
QLabel
{
font-size:22px;
color: rgb(255, 255, 255);
}
QPushButton{
border-width:0;
border-style:outset;
background-color: rgb(80, 80, 80);
font-size:18px;
color:rgb(255, 255, 0);
}
QPushBUtton::pressed{
background-color: rgb(120, 120, 120);
}
QPushButton::checked{
color:red;
}
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>驱动板旧号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>驱动板新号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>390</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>注：直接设定驱动板号即可</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_old">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>120</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-width:0;
border-style:outset;
background: transparent;
font-size:20px;
color:white;}
QPushButton::checked{
color:red;
}
</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_new">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>170</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-width:0;
border-style:outset;
background: transparent;
font-size:20px;
color:white;}
QPushButton::checked{
color:red;
}
</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_set">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>设定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>220</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sub">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>220</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>380</y>
     <width>51</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-width:0;
border-radius:10;
background-color: rgb(80, 80, 80);
font-size:18px;
color:rgb(255, 255, 0);
}
QPushBUtton::pressed{
background-color: rgb(120, 120, 120);
}
</string>
   </property>
   <property name="text">
    <string>返
回</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>701</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 255, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:yellow;
font-size:28px;</string>
   </property>
   <property name="text">
    <string>菜单</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="widget_test" native="true">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>381</width>
     <height>131</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton_stop">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_testnum">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-width:0;
border-style:outset;
background: transparent;
font-size:20px;
color:white;}
QPushButton::checked{
color:red;
}
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_test">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>测试</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>驱动板测试号：</string>
    </property>
   </widget>
  </widget>
  <zorder>widget_test</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_4</zorder>
  <zorder>pushButton_old</zorder>
  <zorder>pushButton_new</zorder>
  <zorder>pushButton_set</zorder>
  <zorder>pushButton_cancel</zorder>
  <zorder>pushButton_add</zorder>
  <zorder>pushButton_sub</zorder>
  <zorder>pushButton_back</zorder>
  <zorder>line_2</zorder>
  <zorder>label_title</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
